frontend/packages/noobaa-storage-plugin/src/components/status-card/status-card.tsx

```diff
@@ -0,0 +1,157 @@
+
+  return (
+    <AlertsBody
+      isLoading={!alertsResponse}
+      error={alertsResponseError}
+      emptyMessage="No object service alerts"
+    >
+      {alerts.length > 0 &&
+        alerts.map((alert) => <AlertItem key={alertURL(alert, alert.rule.id)} alert={alert} />)}
+    </AlertsBody>
+  );
+});
+
+const StatusCard: React.FC<DashboardItemProps> = ({
+  watchK8sResource,
+  stopWatchK8sResource,
+  watchPrometheus,
+  resources,
+  stopWatchPrometheusQuery,
+  prometheusResults,
+}) => {
+  React.useEffect(() => {
+    watchK8sResource(noobaaResource);
+    statusCardQueries.forEach((key) => watchPrometheus(StatusCardQueries[key]));
+    watchPrometheus(DATA_RESILIENCE_QUERIES.REBUILD_PROGRESS_QUERY);
+    return () => {
+      stopWatchK8sResource(noobaaResource);
+      statusCardQueries.forEach((key) => stopWatchPrometheusQuery(StatusCardQueries[key]));
+      stopWatchPrometheusQuery(DATA_RESILIENCE_QUERIES.REBUILD_PROGRESS_QUERY);
+    };
+  }, [watchK8sResource, stopWatchK8sResource, watchPrometheus, stopWatchPrometheusQuery]);
+
+  const buckets = prometheusResults.getIn([
+    StatusCardQueries.BUCKETS_COUNT,
+    'data',
+  ]) as PrometheusResponse;
+
+  const unhealthyBuckets = prometheusResults.getIn([
+    StatusCardQueries.UNHEALTHY_BUCKETS,
+    'data',
+  ]) as PrometheusResponse;
+
+  const pools = prometheusResults.getIn([
+    StatusCardQueries.POOLS_COUNT,
+    'data',
+  ]) as PrometheusResponse;
+
+  const unhealthyPools = prometheusResults.getIn([
+    StatusCardQueries.UNHEALTHY_POOLS,
+    'data',
+  ]) as PrometheusResponse;
+
+  const progressResult = prometheusResults.getIn([
+    DATA_RESILIENCE_QUERIES.REBUILD_PROGRESS_QUERY,
+    'data',
+  ]) as PrometheusResponse;
+  const progressError = prometheusResults.getIn([
+    DATA_RESILIENCE_QUERIES.REBUILD_PROGRESS_QUERY,
+    'loadError',
+  ]);
+
+  const noobaa = _.get(resources, 'noobaa') as FirehoseResult;
+
+  const queriesLoadError = statusCardQueries.some((q) => {
+    return prometheusResults.getIn([StatusCardQueries[q], 'loadError']);
+  });
+
+  const hasStatusLoadError = _.get(noobaa, 'loadError') || queriesLoadError;
+  const allStatusLoaded =
+    _.get(noobaa, 'loaded') && buckets && unhealthyBuckets && pools && unhealthyPools;
+
+  const objectServiceState: ObjectServiceState = getNooBaaState(
+    [buckets, unhealthyBuckets, pools, unhealthyPools],
+    !!hasStatusLoadError,
+    !allStatusLoaded,
+    noobaa,
+  );
+
+  const dataResiliencyState: ObjectServiceState = getDataResiliencyState(
+    [progressResult],
+    !!progressError,
+    !progressResult,
+  );
+
+  return (
+    <DashboardCard>
+      <DashboardCardHeader>
+        <DashboardCardTitle>Status</DashboardCardTitle>
+      </DashboardCardHeader>
+      <DashboardCardBody>
+        <HealthBody>
+          <Gallery className="nb-status-card__health" gutter="md">
+            <GalleryItem>
+              <HealthItem
+                title="Multi Cloud Object Gateway"
```

Comment:
Is it Multi Cloud Object Gateway or Multi Cloud Gateway?

Reply:
Multi Cloud Object Gateway

Reply:
https://docs.google.com/document/d/1Vd3e9YTNczRap18UxjGW_FXap7GhGJ14XE7oPoAGg2g/edit?ts=5d99beba these documents have it as Multi-Cloud Gateway. Could you verify with UX designers so that there's consistency throughout the console?

Reply:
It is what you are seeing

Reply:
Agree that it needs to be consistent.
But this name is considered after discussion with PM.
Can you pinpoint me the place where it is, I can take it forward, once NooBaa team comes back.